src/System.Text.Json/tests/JsonNodeTestData.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace System.Text.Json.Tests
+{
+    /// <summary>
+    /// Helper class simulating external library
+    /// </summary>
+    internal static class EmployeesDatabase
+    {
+        private static int s_id = 0;
+        public static KeyValuePair<string, JsonNode> GetNextEmployee()
+        {
+            var employee = new JsonObject()
+            {
+                { "name", "John" } ,
+                { "surname", "Smith"},
+                { "age", 45 }
+            };
+
+            return new KeyValuePair<string, JsonNode>("employee" + s_id++, employee);
+        }
+
+        public static IEnumerable<KeyValuePair<string, JsonNode>> GetTenBestEmployees()
+        {
+            for (int i = 0; i < 10; i++)
+                yield return GetNextEmployee();
+        }
+
+        /// <summary>
+        /// Returns following JsonObject:
+        /// {
+        ///     { "name" : "John" }
+        ///     { "phone numbers" : { "work" :  "123-456-7890", "home": "123-456-7890"  } }
+        ///     { 
+        ///         "reporting employees" : 
+        ///         {
+        ///             "software developers" :
+        ///             {
+        ///                 "full time employees" : /JsonObject of 3 employees fromk database/ 
+        ///                 "intern employees" : /JsonObject of 2 employees fromk database/ 
+        ///             },
+        ///             "HR" : /JsonObject of 10 employees fromk database/ 
+        ///         }
+        /// </summary>
+        /// <returns></returns>
+        public static JsonObject GetManager()
+        {
+            var manager = GetNextEmployee().Value as JsonObject;
+
+            manager.Add
+            (
+                "phone numbers",
+                new JsonObject()
+                {
+                    { "work", "123-456-7890" }, { "home", "123-456-7890" }
```

Comment:
While this is an invalid phone number it's only because Nigerian phone numbers all have a "1" after the 4.  `425-555-01xx` is a better choice (`-555-01` is reserved)

Reply:
Ok, I can change this...